src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -146,7 +145,7 @@ public ObservableList<Person> getPersonList() {
 
     @Override
     public ObservableList<Tutorial> getTutorialList() {
-        return FXCollections.observableList(tutorials);
+        return FXCollections.unmodifiableObservableList(tutorials);
```

Comment:
```suggestion
        return tutorials.asUnmodifiableObservableList();
```
Follow the `persons` field

Reply:
Does not work as `asUnmodifiableObservableList` is a custom method in `UniquePersonList` class